dance/transforms/filter.py

```diff
@@ -1053,6 +1053,17 @@ def __init__(self, layer: Optional[str] = None, n_top_genes: Optional[int] = 100
                          flavor=flavor, subset=subset, inplace=inplace, batch_key=batch_key, **kwargs)
 
 
+@register_preprocessor("filter", "gene")
+class FilterGenesPlaceHolder(BaseTransform):
+    """Used as a placeholder to skip the process."""
```

Comment:
Used as a placeholder to skip the process.

Reply:
This can stay for now, but will need to be removed eventually.

I will implement a more general bypass mechanism from the pipeline object so that there's always an option to bypass a specific step.